module/easter_egg_func.py

```diff
@@ -44,3 +48,60 @@ def lei_che_ne_pensa_signorina(update: Update, context: CallbackContext) -> None
     check_log(update, context, "leiCheNePensaSignorina")
     message_text = EasterEgg.get_lei_che_ne_pensa_signorina()
     context.bot.sendMessage(chat_id=update.message.chat_id, text=message_text)
+
+
+def tomarchio_schedule(context: CallbackContext) -> None:
+    """Called once a day at 00:00 (ROME)
+    Schedules the "tomarchio_request" job that will run between 10:00 and 21:00 of the same day
+    Also, makes sure the file "data/soldini" exists
+
+    Args:
+        context (CallbackContext): context passed by the updater
+    """
+    if not path.exists("data/soldini"):
+        with open("data/soldini", 'w+') as soldini:
+            soldini.write("0")
+    #context.job_queue.run_once(tomarchio_request, when=time(hour=randint(10, 21), minute=randint(0, 59)))
+    context.job_queue.run_once(tomarchio_request, when=0)
+
+
+def tomarchio_request(context: CallbackContext) -> None:
+    """Called every day between 10:00 and 21:00
+    Sends a message to a random chat choosen between the ones listed in "easter_eggs_chat" in the settings.yaml file
+
+    Args:
+        context (CallbackContext): context passed by the handler
+    """
+    chat_id = choice(config_map['easter_eggs_chat'])
+    message = context.bot.sendMessage(chat_id=chat_id, text="Hai un euro per il Signor Tomarchio?")
+    sleep(30)
```

Comment:
mi spaventa questo sleep, alla fine è in una jobqueue, quindi non dovrebbe bloccare l'intero bot per 30 secondi, ma solo questo thread/processo, ma stiamo attenti  testiamo bene questa feature

Reply:
Si, i job sono eseguiti in un thread, quindi non dovrebbe dare problemi al bot. Comunque si, sarebbe sempre da testare tutto, anche solo che funzioni come previsto